test/test_jit.py

```diff
@@ -358,6 +358,16 @@ def fn(x, y):
         self.assertExpectedGraph(trace)
         self.assertExportImport(trace, (x, y))
 
+    def test_scalar(self):
+        # NB: must not require grad; if it requires grad, it's always a Tensor
+        x = torch.tensor(2.)
+        y = torch.tensor(3.)
+
+        def fn(x, y):
+            return x - y
+        trace, _ = torch.jit.get_trace_graph(fn, (x, y), nderivs=0)
+        print(trace)
```

Comment:
do you want to print the trace in the test?  It's fine, just curious.

Reply:
Nope, it's an accident